src/main/java/org/chirvascristina/classesandobjects/autoservicetask/Person.java

```diff
@@ -0,0 +1,7 @@
+package org.chirvascristina.classesandobjects.autoservicetask;
+
+public class Person {
+    public String phoneNumber;
+    public String owner;
+
```

Comment:
constructorul din main de tip Person "client1" sa fie adaugat in clasa respectiva